helix-contract/contracts/xtoken/v3/base/XTokenBacking.sol

```diff
@@ -136,17 +142,19 @@ contract XTokenBacking is XTokenBridgeBase {
         address _originalToken,
         address _originalSender,
         address _recipient,
+        address _rollbackAccount,
         uint256 _amount,
         uint256 _nonce,
         bytes memory _extParams
     ) external payable {
-        require(_originalSender == msg.sender || _recipient == msg.sender || dao == msg.sender, "invalid msgSender");
-        bytes32 transferId = getTransferId(_nonce, block.chainid, _remoteChainId, _originalToken, _originalSender, _recipient, _amount);
+        require(_rollbackAccount == msg.sender || dao == msg.sender, "invalid msgSender");
```

Comment:
What is rollbackAccount?

Reply:
`_originalSender` is the msg.send who call the backing contract. And the `_rollbackAccount` is configured by user who start the cross-chain transfer.

Sometime if we use a contract to call backing transfer, such as WETH convertor, the `_originalSender` is this convert contract, and it can't call the rollback function on target chain.